### PR TITLE
Add github ribbon

### PR DESCRIPTION
**Pull request by iamneha**
screen shot 2018-05-02 at 2 01 26 am

### CODE REVIEW

**Comment by iamneha** [comment]
#10

**Comment by sara-02** [comment]
@iamneha please go ahead and add changes in other files too.

File: index.html [hunk 1]
```diff
@@ -19,18 +19,16 @@
...
   <body>
 
     <!-- Navigation -->
     <nav class="navbar navbar-expand-lg navbar-light bg-faded py-lg-4">
       <div class="container">
+      <a href="https://github.com/Diversifyingopensource/diversifyingopensource.github.io"><img style="position: absolute; top: 0; left: 0; border: 0;" src="https://s3.amazonaws.com/github/ribbons/forkme_left_gray_6d6d6d.png" alt="Fork me on GitHub"></a>
         <a class="navbar-brand text-uppercase text-expanded font-weight-bold d-lg-none" href="#">Start Bootstrap</a>
-        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
-          <span class="navbar-toggler-icon"></span>
-        </button>
```
Comment (sara-02) [inline index.html]: @iamneha Why did we delete this?

File: index.html [hunk 2]
```diff
@@ -65,6 +63,7 @@ <h2 class="text-center text-lg text-uppercase my-0">Diversifying
         <p>This is an experimental project and will be active from December 2017 to December 2018. We are based out of Bangalore, India. Look out this space for the meetup details.</p>
 
       </div>
+    </div>
```
Comment (sara-02) [inline index.html]: @iamneha Extra div tag? or did we miss this in the previous one?

File: index.html (hunk 1 above)
Comment (iamneha) [reply index.html]: This button is not used anywhere. It was throwing the error in my env as well. Unnecessary code

File: index.html (hunk 2 above)
Comment (iamneha) [reply index.html]: missed this in the previous one

File: index.html (hunk 1 above)
Comment (Rakhisharma) [reply index.html]: @iamneha thanks for the pull request. About the button- We are using this button! If you open the same website on your mobile, you can see this button on `top right-hand side` as navbar button.

Comment (iamneha) [reply index.html]: @Rakhisharma cool I am putting this back then. But I don't think we should show "Start Bootstrap" on mobile, there is no sense of showing it. There are some issues on other pages as well so I'll open different issues.

File: index.html (hunk 2 above)
Comment (sara-02) [reply index.html]: Ok.

**Comment by sara-02** [comment]
@iamneha Any update on the PR?

**Comment by iamneha** [comment]
@sara-02 From my side I think this is done. this PR is only for GitHub ribbon.

**Comment by iamneha** [comment]
It's been months and I think this PR is not going anywhere.
@Rakhisharma can we close this?

**Comment by Rakhisharma** [comment]
Thanks for the PR Neha, Apology for the delay!

**Pull request merged**